doc/source/releasenotes/1.24.0.rst

```diff
@@ -0,0 +1,89 @@
+===================
+1.24.0 "Aquadonkey"
+===================
+
+--------------------
+Upgrade Instructions
+--------------------
+
+This feature release requires a database schema upgrade. Please consult the `documentation <https://rucio.readthedocs.io/en/latest/database.html>`_ about upgrading your database schema.
+
+The following changes are necessary and are covered by the ``alembic upgrade head`` command:
+
+1. Adding missing indices which were only available as functional oracle indices. For oracle databases installed with the `schema.sql <https://github.com/rucio/rucio/blob/master/etc/sql/oracle/schema.sql>`_ definition, these should already exist (Please verify). (Alembic revision ``8ea9122275b1``)
+
+  - Adding ``SUBSCRIPTIONS_STATE_IDX`` on ``subscriptions`` table
+  - Adding ``CONTENTS_RULE_EVAL_FB_IDX`` on ``contents`` table
+  - Adding ``REPLICAS_STATE_IDX`` on ``replicas`` table
+  - Adding ``BAD_REPLICAS_ACCOUNT_IDX`` on ``bad_replicas`` table
+  - Adding ``REQUESTS_DEST_RSE_ID_IDX`` on ``requests`` table
+
+2. Update of ``REQUESTS_STATE_CHK`` constraint in ``requests`` table in preparation for Conveyor Preparer (Alembic revision ``d23453595260``)
+
+-----------------
+Important changes
+-----------------
+
+1. This Rucio release line drops support for Python 2.7 on the Server/Daemon Side. Only Python 3.6, 3.7 and 3.8 are supported from now on. Rucio clients are still Python 2.7 compatible for the foreseen future (EOL will be announced). If your environment is requiring to run Rucio under Python 2.7 we recommend the Rucio 1.23 LTS release line, which will be supported until 2022.
```

Comment:
Not sure if we should announce server/daemon compatibility with Python 3.8 yet, since this is untested.

Reply:
I am overly optimistic :-D
But it's a good point, I will take it out for now.